case-studies/antipatterns/classic/app/pods/two-way-binding/controller.js

```diff
@@ -17,7 +17,7 @@ export default Controller.extend({
 
   actions: {
     setName(event) {
-      this.set('name', event.target.value);
+      this.set('person.name', event.target.value);
```

Comment:
You have successfully avoided using the two-way data binding, but the `name` property on the controller is still using the `alias` CP macro.

The goal is to switch it to `readOnly`, after which the change that you have made would be a necessity to resolve an error.